### PR TITLE
add runtime.getContexts

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Content in support of [Bug 1875480](https://bugzilla.mozilla.org/show_bug.cgi?id=1875480) Implement runtime.getContexts

#### Related issues

MDN documentation PR https://github.com/mdn/content/pull/34008

### CODE REVIEW

File: webextensions/api/runtime.json [hunk 1]
```diff
@@ -594,6 +594,26 @@
             }
           }
         },
+        "getContext": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getContext",
+            "support": {
+              "chrome": {
+                "version_added": "116"
```
Comment (rebloor) [inline webextensions/api/runtime.json:602]: As per https://developer.chrome.com/docs/extensions/reference/api/runtime#method-getContexts

File: webextensions/api/runtime.json [hunk 2]
```diff
@@ -594,6 +594,26 @@
...
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "127"
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
```
Comment (rebloor) [inline webextensions/api/runtime.json:611]: Unable to find mention of implementation in release notes (via Google search)

**Comment by Juraj-Masiar** [comment]
Hello,
Any idea why is the compat data still not available on MDN?
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getContexts#browser_compatibility

**Comment by rebloor** [comment]
@Juraj-Masiar its a typo - raising a PR

**Pull request merged**